test/sdk.test.ts

```diff
@@ -161,11 +161,7 @@ describe('SDK Init', () => {
 
     init({ enabled: true }, (capacitorOptions: CapacitorOptions) => {
       expect(capacitorOptions).toBeDefined();
-      const expectedName = 'Capacitor RewriteFrames';
-      const rewriteFrames = (capacitorOptions.defaultIntegrations as Integration[]).find(function (integration) {
-        return integration.name === expectedName;
-      }) as Integration;
-      expect(rewriteFrames.name).toBe(expectedName);
+      expect(capacitorOptions.integrations).toContain(capacitorRewriteFramesIntegration);
```

Comment:
This seems of, integrations is an array of instantiated integrations, this checks if the array contains the factory function.

Reply:
The test itself only validates if the factory was added on the SDK, if it was not, you get a similar error like 
   ```
Expected value: [Function capacitorRewriteFramesIntegration]
    Received array: [[Function nativeReleaseIntegration], [Function eventOriginIntegration], [Function sdkInfoIntegration]]
```

the iterator tests were added on test/integrations/rewriteframes.test.ts

Reply:
Yes, thank you I see the rewrite tests.

But the issue is that the integrations array is array of integrations (https://github.com/getsentry/sentry-javascript/blob/874df8e5b28e2f0f530b0e651067cdd2aa3e5cf7/packages/types/src/integration.ts#L15) but here it's list of the factory functions, it doesn't break as Function has property `name` and integration has only one required property `name`, but that's just unfortunate coincidence.

Reply:
I got it. I have refactored this test to be more precise and validate a simple capacitor stackframe.